src/services/conversionRates/getConversionRatesService.js

```diff
@@ -24,6 +24,56 @@ const _getRatesDb = async (app, timestamp, symbol = 'ETH') => {
   return { timestamp, rates: {} };
 };
 
+/**
+ * Get the rates from the coingecko API
+ *
+ * @throws Error if fetching the rates from cryptocompare API failed
+ *
+ * @param {Number} timestamp   Timestamp for which the value should be retrieved
+ * @param {Array}  ratestToGet Rates that are missing in the DB and should be retrieved
+ * @param {Array}  stableCoins coins whose value equal one usd
+ *
```

Comment:
These lines are copied from `_getRatesCryptocompare` function doc and are not correct

Reply:
Fixed this block